ReelTalk/containers/MovieDetailView.js

```diff
@@ -93,6 +93,7 @@ var MovieDetailView = React.createClass({
 var styles = StyleSheet.create({
     scrollView: {
       flex: 1,
+      marginBottom: 50
```

Comment:
I think we can avoid doing this with automaticallyAdjustContentInsets={true} in the scrollView